pkgs/servers/sql/postgresql/ext/pljava.nix

```diff
@@ -0,0 +1,81 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  openssl,
+  openjdk,
+  maven,
+  postgresql,
+  libkrb5,
+  makeWrapper,
+  gcc,
+  pkg-config,
+  which,
+}:
+
+maven.buildMavenPackage rec {
+  pname = "pljava";
+
+  version = "1.6.7";
+
+  src = fetchFromGitHub {
+    owner = "tada";
+    repo = "pljava";
+    rev = "V${lib.replaceStrings [ "." ] [ "_" ] version}";
+    sha256 = "sha256-M17adSLsw47KZ2BoUwxyWkXKRD8TcexDAy61Yfw4fNU=";
+
+  };
+
+  mvnParameters = "clean install -Dmaven.test.skip -DskipTests -Dmaven.javadoc.skip=true";
+
+  mvnHash = "sha256-lcxRduh/nKcPL6YQIVTsNH0L4ga0LgJpQKgX5IPkRzs=";
+
+  nativeBuildInputs = [
+    makeWrapper
+    maven
+    openjdk
+    postgresql
+    openssl
+    gcc
+    libkrb5
+    pkg-config
+  ];
+
+  buildInputs = [
+    stdenv.cc.cc.lib
+    which
+  ];
+
+  preBuild = ''
+    export PATH=$(lib.makeBinPath [ postgresql ]):$PATH
+  '';
+
+  buildOffline = true;
+
+  installPhase = ''
+    mkdir -p $out/pljavabuild
+    cp -r *   $out/pljavabuild
```

Comment:
This dance of moving it into a transient directory in `$out` first and then moving it to the final places seems a little odd.
Any specific reason this is done instead of copying everything to the correct place in the first place?